datawig/column_encoders.py

```diff
@@ -440,19 +440,24 @@ def __init__(self,
                  output_column: str = None,
                  max_tokens: int = 2 ** 18,
                  tokens: str = 'chars',
+                 ngram_range: tuple = None,
                  prefixed_concatenation: bool = True) -> None:
 
         ColumnEncoder.__init__(self, input_columns, output_column, int(max_tokens))
 
+        if ngram_range is None:
+            ngram_range = (1, 3) if tokens == 'words' else (1, 5)
+
         if tokens == 'words':
-            self.vectorizer = TfidfVectorizer(max_features=int(self.output_dim), ngram_range=(1, 3))
+            self.vectorizer = TfidfVectorizer(max_features=int(self.output_dim), ngram_range=ngram_range)
         elif tokens == 'chars':
-            self.vectorizer = TfidfVectorizer(max_features=int(self.output_dim), ngram_range=(1, 5),
-                                                analyzer="char")
+            self.vectorizer = TfidfVectorizer(max_features=int(self.output_dim), ngram_range=ngram_range,
+                                              analyzer="char")
         else:
             logger.info(
                 "BowEncoder attribute tokens has to be 'words' or 'chars', defaulting to 'chars'")
-            self.vectorizer = TfidfVectorizer(max_features=int(self.output_dim), ngram_range=(1, 5), analyzer="char")
+            self.vectorizer = TfidfVectorizer(max_features=int(self.output_dim),
```

Comment:
not sure, maybe the int casting as part of the constructor?

Reply:
yeah, good call.